opta/commands/apply.py

```diff
@@ -334,6 +326,39 @@ def _apply(
         )
 
 
+def _verify_semver(old_semver_string: str, current_semver_string: str) -> None:
+    if old_semver_string not in [DEV_VERSION, ""] and current_semver_string not in [
+        DEV_VERSION,
+        "",
+    ]:
```

Comment:
```suggestion
    if old_semver_string in [DEV_VERSION, ""] or current_semver_string in [
        DEV_VERSION,
        "",
    ]:
        return
```

Then un-indent the rest of the lines in this function by one level.

Reply:
done